openfisca_us/variables/gov/states/va/tax/deductions/va_age_deduction.py

```diff
@@ -0,0 +1,40 @@
+from openfisca_us.model_api import *
+
+
+class va_age_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "VA age deduction"
+    unit = USD
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        gov = parameters(period).gov
+        va_deductions_path = (
+            gov.states.va.tax.deductions.income_based_age_deduction
+        )
+        age_thd = va_deductions_path.age
+        # number of tax payers claiming an income based deduction
+        filing_status = tax_unit("filing_status", period)
+        age_head = tax_unit("age_head", period)
+        age_spouse = tax_unit("age_spouse", period)
+
+        n_filers = (age_head >= age_thd) + (age_spouse >= age_thd)
+        if n_filers == 0:
```

Comment:
OpenFisca is numpy-based, so you'll need to replace `if` statements with `where`s. In this case, since you're multiplying by the number of aged filers, you can delete these lines.